boston.html

```diff
@@ -16,8 +16,78 @@
 							<h2>Things to do in Boston</h2>
 							<span class="byline">What to do while you're in town</span>
 						</header>
-						<p>You may just be in Boston for a couple day's, but we've lived here for a couple years (and Kelly grew up here).  So here are some of our suggestions for things to do if you have some free time while you're in town.  Unfortunately, they're all in pseudo-latin.</p>
-						<p>Donec nonummy magna quis risus. Quisque eleifend. Maecenas pede nisl, elementum eu, ornare ac, malesuada at, erat. Proin gravida orci porttitor enim accumsan lacinia. Donec condimentum, urna non molestie semper, ligula enim ornare nibh, quis laoreet eros quam eget ante. Aliquam libero. Vivamus nisl nibh, iaculis vitae, viverra sit amet, ullamcorper vitae, turpis. Aliquam erat volutpat. Vestibulum dui sem, pulvinar sed, imperdiet nec, iaculis nec, leo. Fusce odio. Etiam arcu dui, faucibus eget, placerat vel, sodales eget, orci. Donec ornare neque ac sem. Mauris aliquet. Aliquam sem leo, vulputate sed, convallis at, ultricies quis, justo. Phasellus tempor vehicula justo. Aliquam lacinia metus ut elit.</p>
+						<p>You may just be in Boston for a couple day's, but we've lived here for a couple years (and Kelly grew up here).  So here are some of our suggestions for things to do if you have some free time while you're in town.  Unfortunately, some are in pseudo-latin.</p>
+						<h2>North End</h2>
+						<h4>Museum of Science</h4>
+						<p> Technically it's not in the North End, but that is what it is closest too.  This is probably Boston's best museum, and if you hit on this weekend, I'd hit this one</p>

```

Comment:
"if you hit on this weekend, I'd hit this one" - I assume that's supposed to be "hit one", and you're being inconsistent with you/I.

(the first is serious, the second less so)